packages/web-pkg/src/components/FilesList/ContextActions.vue

```diff
@@ -75,6 +77,14 @@ export default defineComponent({
     const { actions: showSharesActions } = useFileActionsShowShares({ store })
     const { actions: openShortcutActions } = useFileActionsOpenShortcut({ store })
 
+    const extensionRegistry = useExtensionRegistry()
+    const extensionContextActions = computed(() => {
+      return extensionRegistry
+        .requestExtensions<ActionExtension>('action')
+        .filter(({ scopes }) => scopes.includes('files.context-menu'))
```

Comment:
I think I only saw `context-menu` defined as a scope, not  `files.context-menu`. How is this supposed to work?